### PR TITLE
Added Project to Organization in new QField Sync

### PR DESCRIPTION
**Pull request by SeqLaz**
Added a third method to upload project to Organizations, directly from local project to QFieldCloud Organization in QField Sync 4.6, also added background to images from cancellation suscription

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-3083 How to upload to Organization with the new version of QField Sync](https://app.clickup.com/t/2192114/QF-3083)

**Review by suricactus — COMMENTED** [review]
@SeqLaz this should be option #1 ;)

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 1]
```diff
@@ -180,7 +180,22 @@ A form will ask you for project name, description and local directory. In the lo
 
 There are three ways to share a project with an organization:
 
-**Option 1: Uploading directly to the organization:**
+**Option 1: Directly convert your local project to an Organization QFieldCloud project:**
```
Comment (suricactus) [inline documentation/get-started/tutorials/get-started-qfc.en.md:183]: Convert or create?

Also above it say "three ways to share", isn't more "three ways to make a project being owned" (needs better English)?

Comment (SeqLaz) [reply documentation/get-started/tutorials/get-started-qfc.en.md:183]: Good review

**Review by suricactus — APPROVED** [review]

**Pull request merged**